keyboards/noll/multisteno/info.json

```
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Noll Electronics LLC",
+    "keyboard_name": "Nolltronics Multisteno",
+    "maintainer": "Noll Electronics LLC",
+    "bootloader": "stm32duino",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "stenography": true
```

This does not work. Please apply the suggested changes, or at least verify that the changes you make on your own produce the expected functionality.
```suggestion
        "steno": true
```

---

So sorry, I just realized I was running my linter in the wrong QMK directory 🤦🏻